exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -387,11 +435,13 @@ private void checkTransactionsCommittedSuccessfully(
   void createBlockWithTransactionWithWrongServiceId() {
     try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
       short wrongServiceId = (short) (TestService.SERVICE_ID + 1);
+      EmulatedNode emulatedNode = testKit.getEmulatedNode();
+      KeyPair emulatedNodeKeyPair = emulatedNode.getServiceKeyPair();
```

Comment:
(here and elsewhere) Why these changes? Doesn't `createBlockWithTransactions` accept any transactions?